src/Uno.Foundation/Metadata/AttributeNameAttribute.cs

```diff
@@ -0,0 +1,16 @@
+namespace Windows.Foundation.Metadata;
+
+/// <summary>
+/// Indicates the name of the attribute.
+/// </summary>
+[AttributeUsage(AttributeTargets.Class)]
+public partial class AttributeNameAttribute : Attribute
+{
+	/// <summary>
+	/// Creates and initializes a new instance of the attribute.
+	/// </summary>
+	/// <param name="A_0">One or more of the enumeration values.</param>
+	public AttributeNameAttribute(string A_0) : base()
```

Comment:
A_0 ?

Reply:
@jeromelaban Weird naming, but that matches UWP per documentation https://learn.microsoft.com/en-us/uwp/api/windows.foundation.metadata.attributenameattribute.-ctor?view=winrt-22621

Reply:
🤷‍♂️🤷‍♂️😂